### ISSUE TITLE
GUI crash on startup on iSRD network with GNOME desktop.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1909]
If you run VisIt 2.6.0 or later on the GNOME desktop on the LLNL iSRD network, the gui will crash on startup. I was able to look at a stack and I saw that the crash was in Qt's rendering of the GTK style. To avoid the GTK style, I added "-style windows" to the command line and then the gui came up fine and everything worked. Version 2.6.0 switched us to a new Qt and it is apparently not too happy with our version of GNOME. The crash does not happen on KDE desktops.

I edited the customlauncher for versions 2.6.{0,1,2} and made the scripts add "-style windows" to the command line so users would not run into the crash.

Inside the Customize() method, I added:

 if self.generalArgs.exe_name == "gui":
     self.generalArgs.arguments = self.generalArgs.arguments + ["-style", "windows"]

Bottom line: We should investigate why GTK style crashes on our GNOME desktops -- or failing that, make the customlauncher changes permanent for LLNL's closed customlauncher.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 03/27/2013 02:05 pm
Original update: 06/20/2013 02:29 pm
Ticket number: 1397
